### ISSUE TITLE
Possible problem with snap in 22.04

### ISSUE THREAD

**Issue by stuartlangridge** [stuartlangridge/ColourPicker#85]
Hi Stuart. 
Pick worked very well for me previously, it's a wonderful tool, I love it. Thanks a lot for creating and keeping it alive! 

Unfortunately since I updated to Ubuntu 22.04 it crashes similarly as soon as I click on the pick color button. The gui breaks and won't change the cursor to the picker magnifying glass. I can't get it back from being minimized, only if I click on the top bar, and select pick color from there. No matter where I click after that, the result is always rgb(0,0,0). 
I installed it from the snap store.

Here's a [video about the behavior](https://youtu.be/z7oBte7u48k).

Also the output of my `journalctl | grep "pick"` 
[pick_error.log](https://github.com/stuartlangridge/ColourPicker/files/8714101/pick_error.log)

_Originally posted by @dns-rs in https://github.com/stuartlangridge/ColourPicker/issues/59#issuecomment-1129677473_

**Comment by stuartlangridge** [comment]
@dns-rs the video you present is unfortunately marked as private, so I can't see it.

Thank you for the logs -- I'm not using 22.04 yet, but I'll check into this and see if there's anything I can work out!

**Comment by dns-rs** [comment]
@stuartlangridge Sorry, the video is now set to public. Hope this helps.

**Comment by stuartlangridge** [comment]
To confirm: this worked before on 20.04?

I am worried that this is the graphics driver issue #59 (and you were correct to file it there!) which I've never been able to replicate (because I have different graphics card hardware) and which is why it's still open. The update to 22.04 would probably have come with new graphics drivers too, which is why that might be a confounding factor.

**Comment by dns-rs** [comment]
@stuartlangridge Yes, it worked perfectly on 20.04. I have used it for about a year.
I got a basic "Intel Corporation HD Graphics 620 (rev 02)" card in it, so I wouldn't be surprised if some new drivers that come with 22.04 would cause the issue. 
Thanks for checking anyways I hope someday there will be a solution, if nothing else, a hardware upgrade on my side.

**Comment by fabd** [comment]
FWIW, I tried this for the first time, on a fairly clean install of Ubuntu 22.04.1 LTS with few tweaks. NVIDIA & X Server session (default for NVIDIA GPU).

After doing `sudo snap install ... `

Tried launching , nothing happens.

So I tried in the CLI and it shows this:

```
(pick-colour-picker:14498): Gtk-WARNING **: 12:10:09.007: cannot open display: :0
```

**Comment by shubhambattoo** [comment]
Facing a similar issue on Ubuntu 22.04. It just crashes as I try to pick a colour.  [log.txt](https://gist.github.com/shubhambattoo/d42f96fb54831c3c0503e2d6bf84d478)
